crates/biome_deserialize/src/string_set.rs

```diff
@@ -77,6 +85,25 @@ impl<'de> Deserialize<'de> for StringSet {
     }
 }
 
+impl IntoIterator for StringSet {
+    type Item = String;
+    type IntoIter = IntoIter<String>;
+
+    fn into_iter(self) -> Self::IntoIter {
+        self.0.into_iter()
+    }
+}
+
+impl Merge for StringSet {
+    fn merge_with(&mut self, other: Self) {
+        self.extend(other)
+    }
+
+    fn merge_in_defaults(&mut self) {
+        // Default is an empty set, so nothing to do.
+    }
```

Comment:
The default implementation already covers this case so that this function can be omitted.

Reply:
There is no default implementation, since I thought it'd be better to be explicit (forgetting to implement it could silently skip merging your type otherwise). But maybe I was being overly cautious, and we can add one :)

Reply:
Sorry I thought that the trait implemented a default behaviour.

If not, then I'm happy with the current implementation